mongocrypt/src/lib.rs

```diff
@@ -38,11 +39,18 @@ impl HasStatus for CryptBuilder {
     }
 }
 
+static CRYPT_LOCK: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
```

Comment:
Can we add a comment indicating why the lock is needed, possibly referencing the libmongocrpyt bug filed for this?

Reply:
Done.